### ISSUE TITLE
Use dynamic wallet (un)loading

### ISSUE THREAD

**Issue by Sjors** [chris-belcher/electrum-personal-server#53]
If you're using BitcoinQT then the watch-only wallet used for the Electron server adds visual clutter. It would be nice if the Python script loaded the wallet at launch and then unloads it on exit.

This wouldn't be appropriate on a pruned node though, because it may be force to rescan beyond the prune horizon.

**Comment by chris-belcher** [comment]
This would mean that starting up EPS would require a rescan through the times when it had been shut down? That's quite a tradeoff.

It's not too hard to add though, it could still be appropriate for some people.

**Comment by Sjors** [comment]
Yes, you'd need a rescan when starting EPS, but if you don't want that, just don't shutdown EPS.
